### PR TITLE
Client - Get output IDs from indexer

### LINKED ISSUE #233: Client - get output IDs from indexer
## Description

Implement 
* `/api/plugin/indexer/addresses/[Alias Address]/outputs` 
* `/api/plugins/indexer/addresses/[NFT Address]/outputs` 

The Alias and NTF addresses must be encoded in hex

### PR DESCRIPTION
**Pull request by matjazv**
# Description of change

Fixes #233 

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Unit tests are in files: `test_outputs_from_alias_address.c` and `test_outputs_from_nft_address.c`

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests using CTest that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/client/api/restful/get_outputs_id.h [hunk 1]
```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __CLIENT_API_RESTFUL_OUTPUTS_FROM_ADDRESS_H__
+#define __CLIENT_API_RESTFUL_OUTPUTS_FROM_ADDRESS_H__
```
Comment (matjazv) [inline src/client/api/restful/get_outputs_id.h]: Use new header guard name.

File: src/client/api/restful/get_outputs_id.c [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+      .host = conf->host, .path = cmd_buffer, .use_tls = conf->use_tls, .port = conf->port};
+
+  byte_buf_t *http_res = NULL;
+
+  if ((http_res = byte_buf_new()) == NULL) {
+    printf("[%s:%d]: OOM\n", __func__, __LINE__);
+    return -1;
+  }
+
+  // send request via http client
```
Comment (matjazv) [inline src/client/api/restful/get_outputs_id.c]: The same comment is two lines bellow. This one is probably unnecessary.

**Review by matjazv — COMMENTED** [review]
Please check if all code is formatted correctly.

**Comment by rebinmannady93** [comment]
> Please check if all code is formatted correctly.

Done.

File: src/client/api/restful/get_outputs_id.c (hunk 2 above)
Comment (rebinmannady93) [reply src/client/api/restful/get_outputs_id.c]: Removed.

File: src/client/api/restful/get_outputs_id.h (hunk 1 above)
Comment (rebinmannady93) [reply src/client/api/restful/get_outputs_id.h]: Done.

**Review by oopsmonk — APPROVED** [review]

**Pull request merged**